Source/DafnyStandardLibraries/examples/DynamicArrayExamples.dfy

```diff
@@ -0,0 +1,52 @@
+module DynamicArrayExamples {
+  import opened DafnyStdLibs.DynamicArray
+  import opened DafnyStdLibs.BoundedInts
+
+  method {:test} InitialCapacity() {
+    var arr := new DynamicArray<int>(0, 101);
+    for i: int := 0 to 100
```

Comment:
`: int` is not needed. It will be inferred.

Reply:
Btw, this test method only pushes 100 elements, so L6 could pass `100` (instead of `101`) to the constructor. In fact, I would argue that using `100` is a better test to make sure the specifications don't have an off-by-1 error somewhere.